javascript/ql/lib/semmle/javascript/Regexp.qll

```diff
@@ -958,6 +958,27 @@ private predicate isUsedAsNonMatchObject(DataFlow::MethodCallNode call) {
   )
 }
 
+/**
+ * Holds if `call` is a call to `search` whose result is used in a way that suggests it returns a number.
```

Comment:
```suggestion
 * Holds if `value` is used in a way that suggests it returns a number.
```

Looks like the docstring is from an earlier iteration of this PR.  

Everything else LGTM 👍